include/zephyr/net/ieee802154.h

```diff
@@ -97,8 +97,23 @@ extern "C" {
  * @{
  */
 
-#define IEEE802154_MAX_PHY_PACKET_SIZE	127 /**< see section 11.3, aMaxPhyPacketSize */
-#define IEEE802154_FCS_LENGTH		2   /**< see section 7.2.1.1 */
+/**
+ * @brief Represents the PHY constant aMaxPhyPacketSize, see section 11.3.
+ *
+ * @note Currently we only support 127 byte sized packets although some PHYs
+ * (e.g. SUN, MSK, LECIM, ...) support larger packet sizes. Needs to be changed
+ * once we want to fully support those PHYs.
+ */
+#define IEEE802154_MAX_PHY_PACKET_SIZE	127
+
+/**
+ * @brief Represents the frame check sequence length, see section 7.2.1.1.
+ *
+ * @note Currently we only support a 2 byte FCS although some PHYs (e.g. SUN,
```

Comment:
I am not sure about the use of "we" (here and in a few other places in this patch, plus in other recent changes too it seems). In other places you have something along the lines of "Currently only XXX is [partially] supported" and I feel it's more appropriate.
I wouldn't let this prevent this PR to go through though.

Reply:
Fixed everywhere (L2, net_mgmt, driver) I found it with `[Ww]e\b`.